requirements.in

```diff
@@ -19,6 +19,7 @@ Flask-OAuthlib
 Authlib~=0.15.3
 Flask-AppBuilder>=3.3.0
 pytz
+pytest
```

Comment:
im surprised we didnt run pytest in our circleci config. Maybe it was removed at some point

Reply:
can you add the running of pytests to our circleci config if it passes?

Reply:
yes, that is a good point. To be honest I was a bit unsure whether I should add pytest or not. Could there be a particular reason why we don't currently have it? Perhaps, @jklukas would you know?

Reply:
To my knowledge, this repo has not historically had any unit tests, meaning there was no need for a pytest dependency. The primary case that we added testing for was whether dags are parseable at all, for which we built the bash script that lists DAGs, thus causing the python code to be loaded and exposing any syntax errors.